Jargon/heatmap.py

```diff
@@ -0,0 +1,8 @@
+def heatmap(df):
```

Comment:
Please review CONTRIBUTING.MD for PR guidelines. Please add docstring and tests.